host/xtest/xtest_uuid_helpers.c

```diff
@@ -6,6 +6,8 @@
 #include <ctype.h>
 #include <stdint.h>
 #include <string.h>
+#include <endian.h>
+#include <openssl/evp.h>
```

Comment:
fix order

Reply:
ok

Reply:
@etienne-lms 
Hmm... I was kinda looking for guidance on how it should be done but best that I could find was reference to clang-format within Linux kernel. And that didn't modify include order.

So are you looking order like this:
```
#include <ctype.h>
#include <endian.h>
#include <openssl/evp.h>
#include <stdint.h>
#include <string.h>
#include <tee_api_types.h>
#include <tee_client_api.h>
#include "xtest_uuid_helpers.h"
```

Reply:
Yes: alphabetical ordering with `<...>` includes first, then alphabetical ordering with `"..."` includes. Thanks.

Reply:
Fixed.